android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -133,6 +135,12 @@ public void onClick(View v) {
                     createNotification();
                     Toast.makeText(SessionDetailActivity.this, R.string.added_bookmark, Toast.LENGTH_SHORT).show();
                 }
+
```

Comment:
@free4murad Can you remove this extra line here and at line 143? It would better if the code is formatted in better way. :)